layouts/page/guideline.html

```diff
@@ -0,0 +1,292 @@
+                        <div
+                            class="box"
+                            style='--bg: url("{{ absURL "guideline/joinus-4.jpg" }}");'
+                        ></div>
+                    </div>
+                </div>
+                <div class="control">
+                    <button id="prev"><span>&ltrif;</span></button>
+                    <button id="next"><span>&rtrif;</span></button>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    {{ partial "footer.html" }}
+
+    {{/* Reference to external css files */}}
+    {{ with resources.Get "guideline.css" | postCSS | minify }}
+        <link rel="stylesheet" href="{{ .Permalink }}" />
+    {{ end }}
+
+
+    <script>
+        // Javascript is used to modify the dropdown's default behavior
+        //  -> Default: Bootstrap's dropdown will open downward, but it will open upward if there is not enough space at the bottom.
+        //     Modify: Dropdown menu only opens downward, even if there is not enough space at the bottom
+        //  -> Default: Bootstrap's dropdown collapses when an item inside the menu is clicked.
+        //     Modify: Dropdown only collapses when clicking outside of it.
+
+        $(document).ready(function () {
+            // Prevent the dropdown from closing when clicking inside it
+            $(".dropdown-menu").on("click", function (event) {
+                event.stopPropagation();
+            });
+
+            // Close the dropdown when clicking outside of it
+            $(document).on("click", function (event) {
+                let target = $(event.target);
+                if (!target.closest(".dropdown, .dropdown-toggle").length) {
+                    $(".dropdown-menu").removeClass("show");
+                }
+            });
+
+            // Open the dropdown when clicking the button
+            $("#dropdownMenuButton").on("click", function (event) {
+                let dropdownMenu = $(this).next(".dropdown-menu");
+                dropdownMenu.toggleClass("show");
+            });
+        });
+
+        // Image carousel 
+
+        var $ = document;
+
+        $.addEventListener("DOMContentLoaded", function () {
+            const sliderMe = () => {
+                let currentPosition = 0,
+                    sliderItem = document.querySelectorAll(".slider-item"),
+                    sliderItemWidth = window
+                        .getComputedStyle(sliderItem[0])
+                        .flexBasis.match(/\d+\.?\d+/g),
+                    sliderInner = $.querySelector(".slider-inner"),
+                    control = {
+                        next: $.querySelector("#next"),
+                        slideNext() {
+                            currentPosition += parseFloat(sliderItemWidth);
+                            if (currentPosition > limitPosition) {
+                                currentPosition = 0;
+                            }
```

Comment:
Yeah, about the image carousel, I actually wanted to just use Bootstrap's carousel, but it doesn't work. I tried so many times, but it still won't work. So, I decided to just use an image carousel I found on CodePen. Let me see what I can do with the button

Reply:
Yeah, I like the auto-scroll feature too! Initially, we planned to have one big image banner like the one on the homepage, but I ended up making it a carousel because I thought it would be better. We should discuss this design decision further. For now, I'll keep the same design with the updated behavior for the prev and next buttons based on your suggestion.